test/verify/check-docker

```diff
@@ -498,5 +498,68 @@ CMD ["/bin/sh"]
         # This should be updated
         b.wait_in_text("#container-details-cpu-row", "1024 shares")
 
+class TestAtomicScan(MachineCase):
+    def testBasic(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("systemctl start docker")
+
+        # HACK put scan results for one of the images into /var/lib/atomic
```

Comment:
No. It's hacky not because it's working around some issue, but because I don't yet know how we can do this in a better way. We should probably make a known-vulnerable image and scan it on every test?